### ISSUE TITLE
Add IPs to firewall without port

### ISSUE THREAD

**Issue by menathor** [cockpit-project/cockpit#11904]
I see there has been some discussion about adding IPs with ports in the firewall. It would be great if we could also add IPs without having to specify a service or port port, i.e. the equivalent of:

firewall-cmd --permanent --zone=public --add-source=xxx.xxx.xxx.xxx/32

There are times when we need to whitelist an IP for all ports

**Comment by martinpitt** [comment]
@Gundersanne, I thought your most recent work allows that now? I. e. through defining a new zone with IP masks, and adding services to that zone?

**Comment by Gundersanne** [comment]
Yes but right now we only show/add/remove zones which have the `default` (= reject) target. So these zones function as a whitelist for the defined services.

But we don't support the most trusted zones, where added services function as a blacklist (and a source address range would function as a whitelist).

So sadly we don't support this (yet).

Also the `public` zone has a `default` target so would never work to whitelist an entire address range.

**Comment by marusak** [comment]
I think this still makes sense, but it would need some good design.

**Comment by Gundersanne** [comment]
We'd need to distinguish between blacklist and whitelist zones.


![image](https://user-images.githubusercontent.com/11140201/67847014-6c7abc00-fb02-11e9-8517-2962a9cdd7bf.png)

**Comment by martinpitt** [comment]
The firewall page has been able to define zones with IP ranges for a long time:

![image](https://user-images.githubusercontent.com/200109/213634789-13b4216e-05ea-4ccb-a7b7-d732a04d8c17.png)

I retitle this issue as it's really confusing otherwise.
